src/main/java/frc/robot/RobotContainer.java

```diff
@@ -134,9 +135,8 @@ private void initSubsystems() {
             {
               System.out.println("Climb enabled");
 
-              climbSubsystem = new ClimbSubsystem();
-              climbUPCommand = new ClimbUPCommand(climbSubsystem);
-              climbDOWNCommand = new ClimbDOWNCommand(climbSubsystem);
+              climbSubsystem = new ClimbSubsystem(driverJoystick2);
+              climbCommand = new ClimbCommand(climbSubsystem);
```

Comment:
Carry-over from Controller-Layouts branch, which isn't merged yet/has feedback. Instead of trying to do both at once, could this PR just have the test mode controller layout for climb? 

For what it's worth here, we're not checking if there are two controllers (e.g. testMode = false) before trying to do this. Could lead to errors or runtime weirdness

Reply:
for now, the plan was to just change it to true for false manually depending one if you had one or two contrllers